src/main/resources/com/google/api/codegen/go/smoke.snip

```diff
@@ -0,0 +1,77 @@
+@extends "go/header.snip"
+@extends "go/initcode.snip"
+
+@snippet generate(view)
+  {@headerComment(view.fileHeader)}
+
+  package {@view.fileHeader.localPackageName}
+
+  import (
+    {@renderImportSection(view.fileHeader.importSection)}
+  )
+
+  import (
+    "cloud.google.com/go/internal/testutil"
+    "golang.org/x/net/context"
+    "google.golang.org/api/iterator"
+    "testing"
+  )
+
+  var _ = iterator.Done
+
+  func {@view.name}(t *testing.T) {
+    if testing.Short() {
+      t.Skip("skipping smoke test in short mode")
+    }
+
+    projectId := testutil.ProjID()
```

Comment:
I have some concerns with the projectId here:
1. projectId is not always needed for every test
2. We should allow the user to pass in the ID from command line. I am not sure what it should look like in Go but in Java we use option args:
https://github.com/googleapis/toolkit/blob/master/src/test/java/com/google/api/codegen/testdata/java_smoke_test_library.baseline#L57